torch/csrc/jit/codegen/cuda/test/test_gpu.cpp

```diff
@@ -25409,6 +25409,36 @@ TEST_F(NVFuserTest, FusionPrint_CUDA) {
   }
 }
 
+TEST_F(NVFuserTest, FusionDropoutError_CUDA) {
```

Comment:
I think we need to add a test asserting that different schedule generates exactly the same result.